ansible_collections/arista/avd/roles/eos_validate_state/templates/generate_state_report_results.j2

```diff
@@ -24,14 +24,14 @@ eos_validate_state_report:
 {% endfor %}
 {% for node in groups[fabric_name] | arista.avd.natural_sort %}
 {# Hardware results - fan status (power supplies) #}
-{%     if copy_hostvars[node].fan_power_supplies_results.results is arista.avd.defined and copy_hostvars[node].power_supplies_results.skipped is not arista.avd.defined %}
-{%         for result in copy_hostvars[node].power_supplies_results.results %}
+{%     if copy_hostvars[node].fan_power_supplies_results.results is arista.avd.defined %}
+{%         for result in copy_hostvars[node].fan_power_supplies_results.results %}
 {%             set test_id.value = test_id.value + 1 %}
 - id: {{ test_id.value }}
   device: "{{ node }}"
   test_category: "Hardware"
   test_description: "Fan status (power supplies)"
-  test: {{ result.powerSupplySlot.label }}
+  test: "Fan power supply {{ result.powerSupplySlot.label }}"
```

Comment:
Same thing here, if it's a power supply fan we are checking it should be "Power supply fan..." imo.